knex/seeds/books_seed.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('book').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('book').insert([
+        {bookcode: 1, title: 'Book 1', publishercode: 11, type: 'FIC'},
+        {bookcode: 2, title: 'Book 2', publishercode: 22, type: 'CMP'},
+        {bookcode: 3, title: 'Book 3', publishercode: 33, type: 'SCI'}
```

Comment:
Can we use actual data for these?

As in actual book titles and stuff.

Reply:
What stuff is already there? And yes, I can do that now. Just wanted to first run it and make sure it all worked well.

Reply:
> What stuff is already there? And yes, I can do that now. Just wanted to first run it and make sure it all worked well.

I mean that a lot of the files, like the knexfile and stuff is inside the scripts/db folder, you shouldn't need to add much more.

Reply:
I was following the tutorial, just looked the best practice was to have those files separated in that way. Looked at a couple other sites too and they seem to all have the migrations, knex folder, seed folder the knex file